BEGINNERS.md

```diff
@@ -54,9 +54,11 @@ follow the [Vagrant setup instructions](SETUP_VAGRANT.md).
 **This is the best and easiest option for all systems including
 GNU/Linux, BSD, Mac OS X, and Windows.**
 
-As an alternative option, we also have instructions for local installation.
-[Debian-based instructions](SETUP_DEBIAN.md) work for Debian or Ubuntu or
-related, and our general [GUIDE](GUIDE.md) has some notes for other systems.
+As an alternative option, we also have instructions for local
```

Comment:
Sorry to be anal, but would you be up for redoing it to keep it to the three lines and keep the original first line unchanged?